### PR TITLE
[WIP] NetworkManager: update to 1.20.0

### PR DESCRIPTION
**Pull request by teldra**

### CODE REVIEW

File: common/shlibs [hunk 1]
```diff
@@ -641,10 +641,10 @@ libip6tc.so.2 iptables-1.8.3_1
...
 libip4tc.so.2 iptables-1.8.3_1
 libipq.so.0 iptables-1.4.7_1
-libnm-util.so.2 libnm-0.9.1.90_1
-libnm-glib.so.4 libnm-0.9.1.90_1
-libnm-glib-vpn.so.1 libnm-0.9.1.90_1
-libnm.so.0 libnm-1.0.0_1
+libnm-util.so.2 libnm-1.18.2_1
+libnm-glib.so.4 libnm-1.18.2_1
+libnm-glib-vpn.so.1 libnm-1.18.2_1
+libnm.so.0 libnm-1.18.2_1
```
Comment (Gottox) [inline common/shlibs]: Is there a reason that this was updated?

Comment (teldra) [reply common/shlibs]: Not yet. Forgot to add WIP.

**Comment by teldra** [comment]
If I try to build libnm-gtk/network-manager-applet with updated NetworkManager, I get
```
configure: error: in `/builddir/network-manager-applet-1.8.22':
configure: error: No package 'NetworkManager' found
No package 'libnm-glib' found
No package 'libnm-util' found
No package 'libnm-glib-vpn' found
```

File: common/shlibs (hunk 1 above)
Comment (teldra) [reply common/shlibs]: The reason to update was this: https://github.com/void-linux/void-packages/pull/13538#issuecomment-519085008
But it did not work.

**Comment by Gottox** [comment]
https://github.com/Gottox/NetworkManager/commit/ec471e66ee14d7da06d4d0a22bc3cdb5f615fd6f
This fixes the drand48_r issue. It currently fails with some ethernet redefinitions.

**Comment by teldra** [comment]
Even with your suggested patches, *-musl fails. Local builds fails with different error. @Gottox

**Comment by Gottox** [comment]
NetworkManager is a beast when it comes to musl ;)

**Comment by teldra** [comment]
This is way over my head.

**Pull request closed without merging**